public/app/features/admin/ServerStats.tsx

```diff
@@ -80,6 +81,12 @@ export const ServerStats = () => {
               { name: 'Organisations', value: stats.orgs },
               { name: 'Users total', value: stats.users },
               { name: 'Active users in last 30 days', value: stats.activeUsers },
+              ...(config.featureToggles.displayAnonymousStats && stats.activeDevices
```

Comment:
Looks like the feature toggle names has changed? Would be good to update it in the PR description.